docs/Reference/API-Objects.md

```diff
@@ -11,47 +11,47 @@ The following objects are parameters for or returned by Besu API methods.
-| **sha3Uncles**       | *Data*, 32&nbsp;bytes | SHA3 of the uncle's data in the block.                             |
-| **logsBloom**        | *Data*, 256 bytes     | Bloom filter for the block logs. `null` when block is pending.     |
-| **transactionsRoot** | *Data*, 32&nbsp;bytes | Root of the transaction trie for the block.                        |
-| **stateRoot**        | Data, 32&nbsp;bytes   | Root of the final state trie for the block.                        |
-| **receiptsRoot**     | Data, 32&nbsp;bytes   | Root of the receipts trie for the block.                           |
-| **miner**            | Data, 20&nbsp;bytes   | Address to pay mining rewards to.                                  |
-| **difficulty**       | Quantity, Integer     | Difficulty for this block.                                         |
-| **totalDifficulty**  | Quantity, Integer     | Total difficulty of the chain until this block.                    |
-| **extraData**        | Data                  | Extra data field for this block. The first 32 bytes is vanity data you can set using the [`--miner-extra-data`](../Reference/CLI/CLI-Syntax.md#miner-extra-data) command line option. Stores extra data when used with [Clique](../HowTo/Configure/Consensus-Protocols/Clique.md#genesis-file) and [IBFT](../HowTo/Configure/Consensus-Protocols/IBFT.md#genesis-file). |
-| **size**             | Quantity, Integer     | Size of block in bytes.                                            |
-| **gasLimit**         | Quantity              | Maximum gas allowed in this block.                                 |
-| **gasUsed**          | Quantity              | Total gas used by all transactions in this block.                  |
-| **timestamp**        | Quantity              | Unix timestamp for block assembly.                                 |
-| **transactions**     | Array                 | Array of [transaction objects](#transaction-object), or 32 byte transaction hashes depending on the specified boolean parameter. |
-| **uncles**           | Array                 | Array of uncle hashes.                                             |
+| Key                  |         Type          | Value                                                                                                                                                                                                                                                                                                                                                                   |
+|:---------------------|:---------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
more whitespace - guessing this is a formatting change - can this be a separate PR?

Reply:
This was caused by my editor automatically trying to make tables easier to view in markdown and I didn't notice it. I reverted the table formatting.